### ISSUE TITLE
Error cannot read config file

### ISSUE THREAD

**Issue by mubbo** [SublimeLinter/SublimeLinter-eslint#257]
Hi Im getting the following error

**Error: cannot read config file c:\dev\\.eslintrc.json**

I have the following config in Sublimelinter.sublime-settings:

```
"eslint": {
                "@disable": false,
                "args": [
                    "-c",
                    "c:/dev/contens5/.eslintrc.json"
                ],
                "excludes": []
            }
```
the path in the error message is not same as in the settings..

**Comment by kaste** [comment]
Hi!

You use forward slashes / here, is there a specific reason for that?

I would first try Windows slashes \\ and then probably enable debug mode in the settings, so we can see if SublimeLinter itself passes the given argument correctly.

**Comment by mubbo** [comment]
backslashes in the sublime settings file are escape characters, its a json file, so if I use a backslash it needs a double backslash... and yes I have tried double backslash, forward slashes, combinations of relative paths..

here is the output from debug mode:
SublimeLinter: eslint: jquery.cms.rowtype.objectcontainer.js ['C:\\dev\\contens5\\node_modules\\.bin\\eslint.cmd', '--format', 'compact', '--stdin', '--stdin-filename', '@', '-c', 'c:\\dev\\contens5\\.eslintrc.json'] 
SublimeLinter: eslint output:
Cannot read config file: C:\dev\.eslintrc.json
Error: Unexpected end of JSON input
SyntaxError: Cannot read config file: C:\dev\.eslintrc.json
Error: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at loadJSONConfigFile (C:\dev\contens5\node_modules\eslint\lib\config\config-file.js:114:21)
    at loadConfigFile (C:\dev\contens5\node_modules\eslint\lib\config\config-file.js:225:26)
    at loadFromDisk (C:\dev\contens5\node_modules\eslint\lib\config\config-file.js:495:18)
    at Object.load (C:\dev\contens5\node_modules\eslint\lib\config\config-file.js:559:20)
    at Config.getLocalConfigHierarchy (C:\dev\contens5\node_modules\eslint\lib\config.js:227:44)
    at Config.getConfigHierarchy (C:\dev\contens5\node_modules\eslint\lib\config.js:179:43)
    at Config.getConfigVector (C:\dev\contens5\node_modules\eslint\lib\config.js:286:21)
    at Config.getConfig (C:\dev\contens5\node_modules\eslint\lib\config.js:329:29)
    at processText (C:\dev\contens5\node_modules\eslint\lib\cli-engine.js:163:33)

**Comment by kaste** [comment]
Hm, this will probably hard to reproduce. You're using the old SublimeLinter3 with an old eslint plugin. That being said, I used your config (with forward slashes, yay) and it correctly erred

```
SublimeLinter: #24 linter.py:66: ERROR: Cannot read config file: c:\dev\contens5\.eslintrc.json
``` 

bc obviously I don't have that file but the path it shows is the same I passed in. (Except that it magically replaced forward to backward slashes.)

If you also use an old eslint itself, look at the `eslint.cmd` file bc it can easily be the source of such errors. AFAIK there are no workarounds possible with the old SublimeLinter version.
